README.md

```diff
@@ -3,6 +3,10 @@
 A spike for a tool to reveal the individual and team contributions that employees have made to its projects.
 
 
+## Getting started
```

Comment:
@RobAWilkinson - I think this commit somehow got clobbered.  I am not seeing it on master?

Reply:
@kfrz - when he merged it into your base branch, it might have been removed.  Can you double check what did/did not make it into master?